src/unix/linux/disk.rs

```diff
@@ -356,6 +356,12 @@ fn get_all_list(container: &mut Vec<Disk>, content: &str, refresh_kind: DiskRefr
 
     let procfs_disk_stats = disk_stats(&refresh_kind);
 
+    // The list of unmounted disks, initially it include all disks
+    let mut unmounted = procfs_disk_stats
+        .keys()
+        .map(|disk| "/dev/".to_owned() + disk)
```

Comment:
should this be a const?

Reply:
The `/dev/` part? I suppose it could be.

Reply:
Yes.